frontend/pulse3d/src/components/layouts/DashboardLayout.js

```diff
@@ -20,16 +20,24 @@ export default function DashboardLayout({ children }) {
 
   useEffect(() => {
```

Comment:
Is this `useEffect necessary anymore now that the one beneath it got added?

Reply:
Yeah I'm trying to think of a better way of doing it, but I need it to get called everything the component mounts, when the account type changes, and when the router changes.

Reply:
because if a user refreshes the page, then it needs to happen on mount too

Reply:
Gotcha